README.rst

```diff
@@ -88,6 +88,15 @@ The micro flavor provides 64MB of RAM, 1 VCPU and 1GB of disk. The micro2 flavor
 same, but with 0GB of disk (i.e. a minimal disk to boot the image is created
 but with barely free space)
 
+Deploying three testbeds
+************************
+
+The *deploythreeglances.py* script deploys three Openstack Image Repositories (Glance) which
+shares the same Identity Service called Keystone. Each glance is deployed in a different server accesible by a
+floating IP. As well as previously, it is possible to access by SSH Key and its configuration
+variables are in the file *config_vars* in */home/ubuntu* folder inside the server.
+These three Glances testbed is used for GlanceSync acceptance tests.
+
```

Comment:
The *deploythreeglances.py* script deploys 3 OpenStack Image Repositories (Glance) which share the same Identity Service called Keystone. Each glance is deployed on a different server accesible by a floating IP. It is possible to access by SSH Key and its configuration variables are in the file *config_vars* in */home/ubuntu* folder inside the server, in the same way that it was explained in "Deploying one testbed" section. These 3 Glance testbeds are used for GlanceSync acceptance tests.